modules/services/auto-upgrade.nix

```
@@ -0,0 +1,94 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.autoUpgrade; in
+
+{
+
+  options = {
+
+    services.autoUpgrade = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to periodically upgrade NixOS to the latest
+          version. If enabled, a systemd timer will run
+          <literal>nixos-rebuild switch --upgrade</literal> once a
+          day.
```

The description does not match the Systemd service.
```suggestion
          Whether to periodically upgrade home-manager to the latest
          version. If enabled, a Systemd timer will run
          <literal>home-manager switch --upgrade</literal> once a
          day.
```